src/main/java/gameboy/gamepack/racinggame/data/entity/Car.java

```diff
@@ -0,0 +1,20 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import gameboy.gamepack.racinggame.data.vo.Position;
+
+public class Car {
+    private Position position;
+
+    public Car() {
+        this.position = new Position();
+    }
+
+    public void run() {
+        position.add();
+    }
+
+
+    public int getPosition() {
```

Comment:
메서드 사이는 한줄 개행이 좋은 것 같습니다.
```suggestion
    }

    public int getPosition() {
```